parlai/mturk/core/dev/test/test_full_system.py

```diff
@@ -415,35 +426,40 @@ def run_task_wait():
         )
 
     def tearDown(self):
-        self.agent_1.always_beat = False
-        self.agent_2.always_beat = False
         for key in self.worlds_agents.keys():
             self.worlds_agents[key] = True
         self.mturk_manager.shutdown()
         self.fake_socket.close()
-        self.task_thread.join()
+        # if self.task_thread.isAlive():
```

Comment:
nit: remove dead code

Reply:
Oops this should very much still be alive. Nice catch 👍